File: Exception_handelling/complex.java
import java.util.*;

public class complex
{
    private int real,img;

    public complex()
    {
        
        real=0;
        img=0;

    }
    public complex(int real,int img)
    {
        this.real=real;
        this.img=img;

    }
    public complex(complex c1)
    {
        real=c1.real;
        img=c1.img;
    }
    public void display()
    {
       System.out.println(real +"+" +img +"i");
    }

    public void modify(complex x)
    {
        x.real=10;
        x.img=20;
    }
}

File: Exception_handelling/complex.cpp
#include<iostream>
using namespace std;

public class complex
{
    private:
            int real,img;

    public:
    
           complex()
           {
               real=10;
               img=23;
           }

           complex(int real,int img)
           {
            this.real=real;
            this.img=img;
           }

           void modify(complex obj){
            obj.real=10;
            obj.img=20;

           }

           public void display()
           {
               cout<<real <<"+"<<img<<"i";
           }
}

int main()
{
    complex c1(1,2);
    complex c2(3,4);
    c1.display();
   
}

        
        
           


          

    
    
    
    
    
    